// check_error 3:25 "too many arguments to 'sum', expected 2"
func sum(a: int, b: int) -> int { return a + b; }
func main() { var foo = sum(1, 2, 3); }
